static/css/style.css

```diff
@@ -1051,6 +1051,27 @@ p {
     font-size: 16px;
 }
 
+.answer_preview{
+    max-height: 500px;
+    overflow-y: scroll;
+    white-space: pre-wrap;
+    border-style: solid;
+    border-color: var(--fg);
+    border-width: 2px;
+    border-radius: 10px;
+    padding-left: 20px;
+}
+
+.answer_preview pre {
+    white-space: pre-wrap;
+    padding: 10px;
+    padding-left: 40px;
+    border-radius: 10px;
+    background-color: var(--code-preview-background);
+    background-clip: padding-box;
+    width: 80%;
+}
+

```

Comment:
Add something like this to make the inline code snippets stand out more:
```
code:not(pre code) {
  background: var(--code-preview-background);
  padding: 1px 4px;
  border-radius: 3px;
}
```

Reply:
i will be redoing this pr for speed etc i will also be adding something like this https://extravi.dev/tailsx/

Reply:
![image](https://github.com/Extravi/tailsx/assets/98912029/57811d3c-a22e-4c1e-99d9-6b6e56873f50)

Reply:
> Add something like this to make the inline code snippets stand out more:
> 
> ```
> code:not(pre code) {
>   background: var(--code-preview-background);
>   padding: 1px 4px;
>   border-radius: 3px;
> }
> ```

It's done thanks